schemas/HPXMLBaseElements.xsd

```diff
@@ -1168,6 +1168,7 @@
 									</xs:element>
 									<xs:element minOccurs="0" name="InteriorFinish" type="InteriorFinishInfo"/>
 									<xs:element minOccurs="0" maxOccurs="1" name="Insulation" type="InsulationInfo"/>
+									<xs:element minOccurs="0" name="MobileHomeBellyOrWing" type="MobileHomeBellyOrWing"/>
```

Comment:
Add some documentation to the FrameFloor to describe that the `ExteriorAdjacentTo` should be `ambient` for mobile home belly and wing.